### ISSUE TITLE
Hwloc mirror - download issue

### ISSUE THREAD

**Issue by DimaK-tracxpoint** [tensorflow/tensorflow#30483]
I am trying to cross-compile TF for Raspberry Pi via Docker container - as described in documentation. Unfortunately. it breaks with the following message:

ERROR: /workspace/tensorflow/core/BUILD:2432:1: no such package '@hwloc//': java.io.IOException: Error downloading [http://mirror.tensorflow.org/download.open-mpi.org/release/hwloc/v2.0/hwloc-2.0.3.tar.gz, https://download.open-mpi.org/release/hwloc/v2.0/hwloc-2.0.3.tar.gz] to /home/dmitry/tensorflow/bazel-ci_build-cache/.cache/bazel/_bazel_dmitry/eab0d61a99b6696edb3d2aff87b585e8/external/hwloc/hwloc-2.0.3.tar.gz: Tried to reconnect at offset 6,391,630 but server didn't support it and referenced by '//tensorflow/core:lib_internal_impl'
ERROR: Analysis of target '//tensorflow/tools/pip_package:build_pip_package' failed; build aborted: no such package '@hwloc//': java.io.IOException: Error downloading [http://mirror.tensorflow.org/download.open-mpi.org/release/hwloc/v2.0/hwloc-2.0.3.tar.gz, https://download.open-mpi.org/release/hwloc/v2.0/hwloc-2.0.3.tar.gz] to /home/dmitry/tensorflow/bazel-ci_build-cache/.cache/bazel/_bazel_dmitry/eab0d61a99b6696edb3d2aff87b585e8/external/hwloc/hwloc-2.0.3.tar.gz: Tried to reconnect at offset 6,391,630 but server didn't support it

Fixes tried with no luck:
- remove a semi-downloaded package from Bazel cache
- download it from the original site and put to a cache
- download from a mirror with Chrome (breaks in two parts)

**System information**
TF_BUILD_INFO = {
container_type: "pi-python3", 
command: "tensorflow/tools/ci_build/pi/build_raspberry_pi.sh PI_ONE", 
source_HEAD: "c407b045b8802f9eded430ef48be18cd85e4788c", 
source_remote_origin: "https://github.com/tensorflow/tensorflow.git", 
OS: "Linux", 
kernel: "4.15.0-54-generic", 
architecture: "x86_64", 
processor: "Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz", 
processor_count: "8", 
memory_total: "16305540 kB", 
swap_total: "16657404 kB", 
Bazel_version: "Build label: 0.24.1", 
Java_version: "1.8.0_222-ea", 
Python_version: "2.7.6", 
gpp_version: "g++ (Ubuntu 4.8.4-2ubuntu1~14.04.4) 4.8.4", 
swig_version: "", 
NVIDIA_driver_version: "418.56", 
CUDA_device_count: "0", 
CUDA_device_names: "", 
CUDA_toolkit_version: ""
}

**Provide the exact sequence of commands / steps that you executed before running into the problem**
CI_DOCKER_EXTRA_PARAMS="-e CI_BUILD_PYTHON=python3 -e CROSSTOOL_PYTHON_INCLUDE_PATH=/usr/include/python3.4"     tensorflow/tools/ci_build/ci_build.sh PI-PYTHON3     tensorflow/tools/ci_build/pi/build_raspberry_pi.sh PI_ONE

**Comment by gadagashwini** [comment]
@DimaK-tracxpoint Please provide us the Tensorflow version. Thanks!

**Comment by DimaK-tracxpoint** [comment]
Checked out r1.14
Commit c407b045b8802f9eded430ef48be18cd85e4788c

**Comment by philippeback** [comment]
I am having the same kind of problem on a RHEL 7.6 box with Anaconda 3, devtoolset-8, and rh-git29.
Bazel is 0.26.1

```
 - /data/p36078/tensorflow/WORKSPACE:19:1
ERROR: An error occurred during the fetch of repository 'hwloc':
   java.io.IOException: Error downloading [http://mirror.tensorflow.org/download.open-mpi.org/release/hwloc/v2.0/hwloc-2.0.3.tar.gz, https://download.open-mpi.org/release/hwloc/v2.0/hwloc-2.0.3.tar.gz] to /root/.cache/bazel/_bazel_root/7e95f03115517fb37af5a5010ad847b8/external/hwloc/hwloc-2.0.3.tar.gz: Read timed out
ERROR: /data/p36078/tensorflow/tensorflow/tools/pip_package/BUILD:155:1: no such package '@hwloc//': java.io.IOException: Error downloading [http://mirror.tensorflow.org/download.open-mpi.org/release/hwloc/v2.0/hwloc-2.0.3.tar.gz, https://download.open-mpi.org/release/hwloc/v2.0/hwloc-2.0.3.tar.gz] to /root/.cache/bazel/_bazel_root/7e95f03115517fb37af5a5010ad847b8/external/hwloc/hwloc-2.0.3.tar.gz: Read timed out and referenced by '//tensorflow/tools/pip_package:licenses'
ERROR: Analysis of target '//tensorflow/tools/pip_package:build_pip_package' failed; build aborted: no such package '@hwloc//': java.io.IOException: Error downloading [http://mirror.tensorflow.org/download.open-mpi.org/release/hwloc/v2.0/hwloc-2.0.3.tar.gz, https://download.open-mpi.org/release/hwloc/v2.0/hwloc-2.0.3.tar.gz] to /root/.cache/bazel/_bazel_root/7e95f03115517fb37af5a5010ad847b8/external/hwloc/hwloc-2.0.3.tar.gz: Read timed out
INFO: Elapsed time: 40.800s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
```

**Comment by DimaK-tracxpoint** [comment]
Temporary workaround is to edit the first URI in third_party/hwloc/workspace.bzl somewhere between domain name and file extension - so it will point to a non-existing location. For example:
...
urls = [
            "http://mirror.tensorflow.org/download.open-mpi.org/release/hwloc/**v2.100**/hwloc-2.0.3.tar.gz",
            "https://download.open-mpi.org/release/hwloc/v2.0/hwloc-2.0.3.tar.gz",
        ],
...
Bazel will pick the second URL and proceed.

**Comment by ParadoxShmaradox** [comment]
I just hit this issue while trying to compile tensorflow 1.14 from source.
I tried to download the file manually from the http://mirror.tensorflow.org site and I got a checkpoint warning of a malicious file and the download was blocked. It's likely a false positive as the checksum of the file from the mirror and the same file from the open-mpi.org site are the same and checkpoint AV doesn't alert on that site.
I also checked other files from the mirror and they download fine.
